it/src/main/java/org/corfudb/universe/node/server/CorfuServerParams.java

```diff
@@ -123,6 +123,6 @@ public static class ContainerResources {
          */
         @Getter
         @Default
-        private final long memory = 1048 * 1024 * 1024;
+        private final long memory = 4L * 1024 * 1024 * 1024;
```

Comment:
4GB for server is higher than many deployments right? Do we want to make this tunable perhaps?

Reply:
@hisundar I set it back to the original value and added a patch to make it tunable: https://github.com/CorfuDB/CorfuDB/pull/3349